src/app/containers/FrontPageSection/index.jsx

```diff
@@ -126,19 +155,45 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
         {group.strapline.name}
       </SectionLabel>
       {items.length > 1 ? (
-        <MarginWrapper firstSection={isFirstSection}>
-          <StoryPromoUl>
-            {items.map((item, index) => (
-              <StoryPromoLi key={item.id}>
-                <StoryPromoComponent
-                  item={item}
-                  sectionNumber={sectionNumber}
-                  storyNumber={index}
-                />
-              </StoryPromoLi>
-            ))}
-          </StoryPromoUl>
-        </MarginWrapper>
+        group.strapline.name === 'Useful links' ? (
+          <UsefulLinksWrapper>
+            <UsefulLinksUl>
+              {items.map(item => {
+                return (
+                  <UsefulLinksLi key={item.id}>
+                    <UsefulLink
+                      script={script}
+                      service={service}
+                      href={item.uri}
+                    >
+                      {item.name}
+                    </UsefulLink>
+                  </UsefulLinksLi>
+                );
+              })}
+            </UsefulLinksUl>
+          </UsefulLinksWrapper>
+        ) : (
+          <MarginWrapper firstSection={isFirstSection}>
+            <StoryPromoUl>
+              {items.map((item, index) => (
+                <StoryPromoLi key={item.id}>
+                  <StoryPromoComponent
+                    item={item}
+                    sectionNumber={sectionNumber}
+                    storyNumber={index}
+                  />
+                </StoryPromoLi>
+              ))}
+            </StoryPromoUl>
+          </MarginWrapper>
+        )
```

Comment:
could this nested ternary be moved out to a separate function?

Reply:
Done :)